### PR TITLE
Fix all examples in apache_os.xml

### PR DESCRIPTION
**Pull request by dabdine**
## Description
Addresses all warnings in `apache_os.xml` using real data from Censys.

## Motivation and Context
Eliminate warnings due to missing examples.

## How Has This Been Tested?
`bundle exec rake tests`

## Types of changes

- Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have updated the documentation accordingly (or changes are not required).
- [x] I have added tests to cover my changes (or new tests are not required).
- [x] All new and existing tests passed.

### CODE REVIEW

File: xml/apache_os.xml [hunk 1]
```diff
@@ -262,48 +295,56 @@
...
+    <example>Apache/2.2.9 (rPath)</example>
     <param pos="0" name="os.vendor" value="rPath"/>
     <param pos="0" name="os.family" value="Linux"/>
     <param pos="0" name="os.product" value="Linux"/>
   </fingerprint>
 
   <fingerprint pattern="\(StartCom Linux\)">
     <description>StartCom Linux</description>
+    <!-- Replace with a useful (real world) example if one is found -->
+    <example>Apache (StartCom Linux)</example>
```
Comment (dabdine) [inline xml/apache_os.xml:339]: I couldn't find an example for this, so I just hand-crafted a string to satisfy the regular expression. StartCom appears to be defunct.

Comment (mkienow-r7) [reply xml/apache_os.xml:339]: I found a mirror that has ISOs for that distro! https://ftp.nluug.nl/pub/os/Linux/distr/startcom/

Comment (dabdine) [reply xml/apache_os.xml:339]: good call!

```
$ wget -q https://ftp.nluug.nl/pub/os/Linux/distr/startcom/AS-5.0.0/os/x86_64/StartCom/RPMS/httpd-2.2.3-31.SEL5_4.x86_64.rpm && \
rpm2cpio httpd-2.2.3-31.SEL5_4.x86_64.rpm | cpio -ivdm ./usr/sbin/httpd && \
strings usr/sbin/httpd | grep StartCom && \
rm httpd-2.2.3-31.SEL5_4.x86_64.rpm
./usr/sbin/httpd
6466 blocks
Apache/2.2.3 (StartCom)
Apache/2.2.3 (StartCom) (Release 31.SEL5_4)
```

Comment (dabdine) [reply xml/apache_os.xml:339]:
```
$ URL="https://ftp.nluug.nl/pub/os/Linux/distr/startcom/ML-5.0.5/os/x86_64/StartCom/RPMS/httpd-2.2.0-5.1.2.x86_64.rpm" FNAME="`basename $URL`" bash -c 'wget -q $URL && rpm2cpio $FNAME | cpio -ivdm ./usr/sbin/httpd && strings usr/sbin/httpd | grep StartCom && rm $FNAME' 
./usr/sbin/httpd
14294 blocks
Apache/2.2.0 (StartCom Linux)
```

**Comment by dabdine** [comment]
Force-pushed to remove an outdated comment without having to mangle history too much.

**Review by mkienow-r7 — APPROVED** [review]
Thank you for the contribution @dabdine!

**Pull request merged**